src/wallet/mod.rs

```diff
@@ -1132,6 +1160,11 @@ where
         descriptor
     }
 
+    /// Returns whether the wallet has a dedicated internal descriptor
+    pub fn has_internal_descriptor(&self) -> bool {
+        self.change_descriptor.is_some()
+    }
```

Comment:
Note you can already just call `wallet.public_descriptor(KeychainKind::Internal).is_some()`